src/jdk.jdwp.agent/unix/native/libjdwp/linker_md.c

```diff
@@ -101,13 +109,13 @@ dbgsysBuildLibName(char *holder, int holderlen, const char *pname, const char *f
     const int pnamelen = pname ? strlen(pname) : 0;
 
     *holder = '\0';
```

Comment:
This assignment is now redundant: please remove it.

Reply:
Done.